crates/sui-types/src/sui_system_state/sui_system_state_inner_v1.rs

```diff
@@ -588,8 +612,14 @@ impl SuiSystemStateTrait for SuiSystemStateInnerV1 {
             storage_fund,
             parameters:
                 SystemParametersV1 {
-                    governance_start_epoch,
+                    stake_subsidy_start_epoch,
                     epoch_duration_ms,
+                    //TODO should we include these in the summary?
```

Comment:
I think so. The explorer / wallet may want to use this info for displaying validator status, etc.